set.mm

```diff
@@ -145394,19 +145426,22 @@ of an open range of nonnegative integers (of length equal to the length of
     hashwrdn $p |- ( ( V e. Fin /\ N e. NN0 )
             -> ( # ` { w e. Word V | ( # ` w ) = N } ) = ( ( # ` V ) ^ N ) ) $=
       ( cfn wcel cn0 wa cv chash cfv wceq cword crab cc0 cfzo cmap cexp wrdnval
-      co fveq2d fzofi a1i hashmap sylan2 hashfzo0 adantl oveq2d 3eqtrd ) CDEZBF
-      EZGZAHIJBKACLMZIJCNBOSZPSZIJZCIJZUMIJZQSZUPBQSUKULUNIABCDRTUJUIUMDEZUOURK
-      USUJNBUAUBCUMUCUDUKUQBUPQUJUQBKUIBUEUFUGUH $.
+      co fveq2d fzofi hashmap mpan2 hashfzo0 oveq2d sylan9eq eqtrd ) CDEZBFEZGZ
+      AHIJBKACLMZIJCNBOSZPSZIJZCIJZBQSZUJUKUMIABCDRTUHUIUNUOULIJZQSZUPUHULDEUNU
+      RKNBUACULUBUCUIUQBUOQBUDUEUFUG $.
 
     $( If there is only a finite number of symbols, the number of words of a
        fixed length over these symbols is also finite.  (Contributed by
-       Alexander van der Vekens, 25-Mar-2018.) $)
-    wrdnfi $p |- ( ( V e. Fin /\ N e. NN0 )
```

Comment:
I still think one should keep this as an OLD version whose hypotheses were reduced.  "Simplify antecedent" could be the reason for the revision.

Reply:
or "Remove unnecessary antecedent". (that text should go right before the "Revised by" tag)